test/manual/templates/run_tests.sh

```
@@ -0,0 +1,24 @@
+# tests/run_tests.sh
+#!/usr/bin/env bash
```

Shellcheck (0.10.0)

[error] The shebang must be on the first line. Delete blanks and move comments.

(SC1128)